test/Dialect/Calyx/emit.mlir

```diff
@@ -614,3 +614,33 @@ module attributes {calyx.entrypoint = "main"} {
     }
   } {toplevel}
 }
+
+// -----
+
+// Emit negative floating point values
+
+module attributes {calyx.entrypoint = "main"} {
+  calyx.component @main(%clk: i1 {clk}, %reset: i1 {reset}, %go: i1 {go}) -> (%done: i1 {done}) {
```

Comment:
Nit: Can we make this test just be 3 constants (one for a negative number, infinity, and a NaN) and as little control flow as possible?

Reply:
Sure, and done!